scripts/ioctool

```diff
@@ -74,10 +153,34 @@ CMD=$2
 if [[ $NAME == *':'* ]]; then #we are assuming that PVs have a colon in them
    iocfpv "$NAME"
    NAME="$ioc"
+
+elif [[ $NAME == *'-'* || $NAME == *'_'* || ^[[:alnum:]]+$ ]]; then
+   INFO=$(grep_ioc "$NAME" all | grep "id:'$NAME'")
+      if [ -z "$INFO" ]; then
+         camname_to_pv "$NAME"
+         iocfpv "$campv"
+         NAME="$ioc"
+        
+      elif [[ -z "$NAME" ]]; then
+         echo "Did not find ${NAME} running anywhere. Exiting..." >&2
+         exit 1
+      fi 
```

Comment:
Please add comments to these ifs and elifs to make it clearer to the next code reader